lib/converter/rule_test.go

```diff
@@ -30,8 +30,12 @@ var (
 	cidr2     = net.MustParseCIDR("11.0.0.1/24")
 	cidr3     = net.MustParseCIDR("12.0.0.1/24")
 	cidr4     = net.MustParseCIDR("13.0.0.1/24")
+	cidr1Net  = net.MustParseCIDR("10.0.0.0/24")
+	cidr2Net  = net.MustParseCIDR("11.0.0.0/24")
 	cidr3Net  = net.MustParseCIDR("12.0.0.0/24")
 	cidr4Net  = net.MustParseCIDR("13.0.0.0/24")
+	cidr1Norm = (&cidr1Net).Network()
+	cidr2Norm = (&cidr2Net).Network()
```

Comment:
Don't know why we have these cidr1-4Norm - they are identical to the cidr1-4Net.

Looks like the tests are wrong I guess.  We should be configuring the API with cidr1-4 and we should validate that the backend model has cidr1-4Net....  and cidr1-4Norm should just be removed.

Reply:
dunno, I just copied what the other tests did for it, I can change it